src/app/page.tsx

```diff
@@ -1,16 +1,66 @@
+'use client'
+import {
+  CodeMarkdown,
+  Container,
+  FormGithub,
+  MarkdownPreview,
+  NavBar,
+} from '@/components'
+import { useEffect, useState } from 'react'
+
 export default function Home() {
+  const [markdown, setMarkdown] = useState('')
+  const [showCodeMarkdown, setshowCodeMarkdown] = useState(false)
```

Comment:
- Es una convensión usar camelCase para el uso de variables en JS
- Se recomienda tipar el useState, solo el tipo de datoa a guardar sería suficiente
- Es una buena practica de legibilidad usar el prefijo `is` siempre y cuando el valor a guardar sea de tipo `boolean`
- Tener cuidado con el uso de inglés `isShowCodeMarkdown `-> `isShowMarkdownCode`

Por ejemplo:
```suggestion
  const [isShowMarkdownCode, setIsShowMarkdownCode] = useState<boolean>(false)
```